### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by BoopPush**
http://pockettrackeritirod.zzz.com.ua/ **(Deploy link)**
Pay attention on animation in diagram, links, buttons, and diagram legend.
Since there is no JS there is no logic inside, but it's temporary.The main goal is to show you UI

### CODE REVIEW

**Comment by nastik1811** [comment]
Hm. And where is mobile version and authorization form?

**Comment by nastik1811** [comment]
And forms for expence/income creation

**Comment by BoopPush** [comment]
There is no mobile version.I forgot about forms I'll add them asap

**Comment by nastik1811** [comment]
How is it "no mobile version"? 👀

**Comment by nastik1811** [comment]
telegram-cloud-photo-size-2-5411633027907436097-y

File: index.html [hunk 1]
```diff
@@ -121,4 +122,63 @@ <h3 class = "child" style="color: #172c66">Transaction</h3>
...
+                <option value="Sport">Sport</option> 
+                <option value="Taxi">Taxi</option> 
+                <option value="Eating out">Eating out</option>
+                <option value="Clubs">Clubs</option>
+                <option value="Clothes">Clothes</option>
+              </select>
+                </span></div>
+                <div><span>Enter ammount: </span><span><input type="text" id="ammount"></span></div>
+                <div><button class="accept">Accept</button></div>
+            </div>
```
Comment (nastik1811) [inline index.html]: it should be a form

File: index.html [hunk 2]
```diff
@@ -121,4 +122,63 @@ <h3 class = "child" style="color: #172c66">Transaction</h3>
...
+          <div class="modal-content">
+            <div class="modal-header">
+              <h3 class="modal-title">Authorization</h3>
+              <a href="#close" title="Close" class="close">×</a>
+            </div>
+            <div class="modal-body"> 
+                <div><span>Enter your name: </span><span><input type="text"></span></div>
+                <div><span>Enter your password: </span><span><input type="text"></span></div>
+                <div><button class="accept">Enter</button></div>
+            </div>
```
Comment (nastik1811) [inline index.html]: also a form

File: index.html [hunk 3]
```diff
@@ -121,4 +122,63 @@ <h3 class = "child" style="color: #172c66">Transaction</h3>
...
+                <option value="">Select an option</option> 
+                <option value="Gifts">Gifts</option> 
+                <option value="Sport">Sport</option> 
+                <option value="Taxi">Taxi</option> 
+                <option value="Eating out">Eating out</option>
+                <option value="Clubs">Clubs</option>
+                <option value="Clothes">Clothes</option>
+              </select>
+                </span></div>
+                <div><span>Enter ammount: </span><span><input type="text" id="ammount"></span></div>
```
Comment (nastik1811) [inline index.html]: "Enter ammount" should be a label, as well as "Choose expense" and so on

**Comment by BoopPush** [comment]
Ok, i see, can i move next?

**Comment by nastik1811** [comment]
Have you fixed comments above?

**Comment by BoopPush** [comment]
Fix comments. Sorry for late,solved the issues with interviews

**Review by nastik1811 — APPROVED** [review]

**Pull request merged**